build.sbt

```
@@ -0,0 +1,18 @@
+enablePlugins(TypelevelCiReleasePlugin)
+
+// Projects
+lazy val `sbt-circe-io` = project.in(file(".")).enablePlugins(NoPublishPlugin).aggregate(core)
+
+lazy val core = project
+  .in(file("core"))
+  .enablePlugins(SbtPlugin)
+  .settings(
+    name := "sbt-circe-io"
```

to match the repo name
```suggestion
    name := "sbt-circe-org"
```

---

Ah, I see. Up for interpretation whether `org` comes from `http4s.org` or just generally means organization. I lean to use org since that's what they're called on GitHub.

---

Indeed, I took `io` from `io.circe` 😅

---

I'm team `sbt-circe-org` because this plugin is for projects in our github org.

---

It's now `sbt-circe-org`, `CirceOrgPlugin` and friends respectively